pages/perfil/index.public.js

```diff
@@ -143,10 +143,9 @@ function EditProfileForm() {
         await fetchUser();
 
         if (user.email !== email) {
-          setErrorObject({
-            message: `Atenção: Um email de confirmação foi enviado para ${email}`,
-            key: 'email',
-            type: 'confirmation',
+          setGlobalMessageObject({
+            type: 'warning',
+            text: `Um email de confirmação foi enviado para ${email}`,
           });
```

Comment:
Precisamos indicar ao usuário que as informações foram salvas, com exceção do e-mail. A verificação `Object.keys(payload).length > 1` é para verificar se o `email` não foi o único valor enviado no `payload`.

Se você pensar numa mensagem melhor, pode trocar. Na condição `false` pensei também na mensagem `Alteração pendente. Um email de confirmação foi enviado para ${email}.`. Não sei se ela fica melhor.

```suggestion
          const hasSavedModifications = Object.keys(payload).length > 1;
          const text = hasSavedModifications
            ? `Alterações salvas. O email será alterado apenas após a confirmação pelo link enviado para ${email}.`
            : `Um email de confirmação foi enviado para ${email}.`;
          setGlobalMessageObject({ text, type: 'warning' });
```

![Screenshot](https://github.com/filipedeschamps/tabnews.com.br/assets/26308880/1458f84d-5776-495b-a5d3-c117aa02ebed)

PS: Se você for alterar a mensagem, não esqueça do ponto final 😅

Reply:
Boa Rafa!

Eu só evitaria colocar o ponto final após o endereço de e-mail. Acho que isso pode causar uma dúvida no usuário se foi ele que inseriu esse ponto sem querer.

Uma alternativa é envolver o endereço com aspas, daí não tem problema colocar o ponto final.

Reply:
Boa!

Atualizei as mensagens aqui, o que acham?

<img width="600" alt="Captura de Tela 2023-12-20 às 22 46 24" src="https://github.com/filipedeschamps/tabnews.com.br/assets/30873873/fa8e48b2-6b64-4257-bfe4-c611ac8875b1">

<img width="600" alt="Captura de Tela 2023-12-20 às 22 47 07" src="https://github.com/filipedeschamps/tabnews.com.br/assets/30873873/0af7cb41-9521-454f-92b5-977592873e25">